### DISCUSSION TITLE
react-medium-image-zoom show high quality image onclick

### DISCUSSION THREAD

**Discussion by levantu28091997** [rpearce/react-medium-image-zoom#460] [General]
I use the library react-medium-image-zoom
My code: 
`
<Zoom classDialog="custom-zoom" zoomImg={{
      src: articleData?.srcset,
      srcSet: articleData?.srcset
    }}  key={`photo-featured`}>
      <ImageMS
        src={articleData?.srcset}
        alt={`teaser image`}
        priority={true}
        service="icms"
        width={1340}
        height={756}
        className={clsx(
          "rounded-lg aspect-video object-cover",
          "w-full mb-2.5",
        )}
      />
    </Zoom>`
Can someone tell me, how can the zoom not exceed the original size of the image?
Like the sample example: https://rpearce.github.io/react-medium-image-zoom/?path=/story/img--small-src-size
Thanks !

**Comment by rpearce** [comment]
@levantu28091997 If you can provide an example I can test, I'm happy to try to help debug it.

**Comment by levantu28091997** [comment]
Thanks @rpearce .
Video: https://www.loom.com/share/fbbf08a6aadb4fa78d89ebfcda3e6b12
You can see in my video that the image has zoomed in beyond the original image size.
We need the same configuration as in your example: https://rpearce.github.io/react-medium-image-zoom/?path=/story/img--small-src-size

**Reply by rpearce** [reply]
@levantu28091997 I meant if you have a working example of the issue that I can see the code for and alter, myself, then I can help.
